docker-compose.dev.yml

```diff
@@ -1,16 +1,11 @@
-version: '3.8'
+version: "3.5"
 
 services:
   db:
     image: postgres:12.3
-    container_name: google_scraper_db_dev
+    container_name: google_scraper_db
     environment:
-      - POSTGRES_DB=${DATABASE_NAME}
+      - POSTGRES_DB=google_scraper_development
       - POSTGRES_HOST_AUTH_METHOD=trust
     ports:
       - "5432:5432"
-    volumes:
-      - pg_data:/var/lib/postgresql/data
-
-volumes:
-  pg_data:
```

Comment:
Don't we need this? 🤔

Reply:
bring it back in [a854eed](https://github.com/carryall/go-google-scraper-challenge/pull/80/commits/a854eed61d0cccd87221fc01c6621ea2c9c1f4f6) 🙈